.ci/Dockerfile.armv7l

```diff
@@ -1,4 +1,4 @@
-ARG image=balenalib/raspberrypi3-debian-python:3.7-buster
```

Comment:
Hi @nidhi-2619 ,

While cross-compiling for RPi, we're targeting specific Python + OS versions to reflect the official Raspbian images.

So, ATM we're processing builds for all the followings:
https://github.com/kivy/kivy/blob/5df8304480f5980ed8dc5f08919efffbe9e48da5/.github/workflows/rpi_wheels.yml#L19

So, since we're moving to `3.8` as a minimum, you should migrate the default to `balenalib/raspberrypi3-debian-python:3.9-bullseye`.

I've added a tag on the PR title to build wheels, so we can test the wheels build when you're ready.